README.md

```diff
@@ -51,6 +61,63 @@ and then run the license-tool:
 yarn licenseCheck:run
 ```
 
+## Branding
+
+Default branding data for the User Dashboard is located in [branding.constant.ts](src/services/bootstrap/branding.constant.ts)#BRANDING_DEFAULT. It can be overridden without re-building the project in [product.json](/assets/branding/product.json). [product.json](/src/assets/branding/product.json) should contain only values that should overwrite default ones.
+
+### Configurability
+
+Configuration defaults for the User Dashboard could be applied in [branding.constant.ts](src/services/bootstrap/branding.constant.ts)#BRANDING_DEFAULT (or updated in [product.json](/src/assets/branding/product.json)) in `"configuration"` section.
+
+Adding the `"configuration.menu.disabled"` field allows Che Admins to list menu entries they want to hide in the left navigation bar, along with the corresponding routes which will be disabled.
+
+Available values are `"administration"`, `"factories"`, `"getstarted"`, `"organizations"`, `"stacks"`.
```

Comment:
Available values for che-dashboard-next should be "getstarted", "workspaces".
```type ConfigurableMenuItem = 'getstarted' | 'workspaces'```
https://github.com/che-incubator/che-dashboard-next/blob/master/src/typings/che.d.ts#L15

Reply:
BTW is it supported in dashboard next?

Reply:
No. It is not fully implemented.

Reply:
I removed that from README until it's not implemented.